Add 2025_Lazaridis_IndoEuropeans

Tlkhi opened the pull request:
### PR Checklist for a new package submission

- [x] The package does not exist already in the community archive, also not with a different name.
- [x] The package title in the `POSEIDON.yml` conforms to the general title structure suggested here: `<Year>_<Last name of first author>_<Region, time period or special feature of the paper>`, e.g. `2021_Zegarac_SoutheasternEurope`, `2021_SeguinOrlando_BellBeaker` or `2021_Kivisild_MedievalEstonia`.
- [x] The package is stored in a directory that is named like the package title.

***

- [x] Samples that already have been published previously, and got re-analysed (e.g. re-sequenced) for the now packaged publication, have a modified `Poseidon_ID` of the form `<Original Poseidon_ID>_<Initials of the main author>_<Year>`. Re-analysed versions of `I1685` (Lazaridis et al. 2016) should, for example, be assigned the IDs `I1685_IL22` (Lazaridis et al. 2022) and `I1685_IL25` (Lazaridis et al. 2025).

***

- [x] The package is complete and features the following elements:
  - [x] Genotype data in binary PLINK format (not EIGENSTRAT format).
  - [x] Genotype has been provided by the original authors of the publication describing the data.
  - [x] A `POSEIDON.yml` file with not just the file-referencing fields, but also the following meta-information fields present and filled: `poseidonVersion`, `title`, `description`, `contributor`, `packageVersion`, `lastModified` (see [here](https://github.com/poseidon-framework/poseidon-schema/blob/master/POSEIDON_yml_fields.tsv) for their definition)
  - [x] A reasonably filled `.janno` file (for a list of available fields look [here](https://github.com/poseidon-framework/poseidon-schema/blob/master/janno_columns.tsv) and [here](https://www.poseidon-adna.org/#/janno_details) for more detailed documentation about them).
  - [x] A `.bib` file with the necessary literature references for each sample in the `.janno` file.
- [x] Every file in the submission is correctly referenced in the `POSEIDON.yml` file and there are no additional, supplementary files in the submission that are not documented there.
- [x] Genotype data, `.janno` and `.bib` file are all named after the package title and only differ in the file extension.
- [x] The package version in the `POSEIDON.yml` file is `1.0.0`.
- [x] The `poseidonVersion` of the package in the `POSEIDON.yml` file is set to the latest version of the [Poseidon schema](https://github.com/poseidon-framework/poseidon-schema/releases).
- [x] The `POSEIDON.yml` file contains the corresponding checksums for the fields `genoFile`, `snpFile`, `indFile`, `jannoFile` and `bibFile`.
- [x] There is either no `CHANGELOG` file or one with a single entry for version `1.0.0`.

***

- [x] The `Publication` column in the `.janno` file is filled and the respective `.bib` file has complete entries for the listed mentioned keys.
- [x] The `.janno` file does not include any empty columns or columns only filled with `n/a`.
- [x] The order of columns in the `.janno` file adheres to the standard order as defined in the Poseidon schema [here](https://github.com/poseidon-framework/poseidon-schema/blob/master/janno_columns.tsv).
- [x] The `.janno` and the `.ssf` files are not fully quoted, so they only use single- or double quotes (`"..."`, `'...'`) to enclose text fields where it is strictly necessary (i.e. their entry includes a TAB).

***

- [x] The package passes a validation with `trident validate --fullGeno`.

***

- [x] Large genotype data files are properly tracked with Git LFS and not directly pushed to the repository. For an instruction on how to set up Git LFS please look [here](https://www.poseidon-adna.org/#/archive_submission_guide?id=submitting-the-package). If you accidentally pushed the files the wrong way you can fix it with `git lfs migrate import --no-rewrite path/to/file.bed` (see [here](https://github.com/git-lfs/git-lfs/blob/main/docs/man/git-lfs-migrate.adoc#import-without-rewriting-history)).

nevrome commented:
Thanks for preparing this - looks very well done at first glance! The only thing I noticed is the `ENA:` in the `Genetic_Source_Accession_IDs` column. The Poseidon schema only requires the ID here.

I'll try to find a reviewer for the package.

nevrome commented:
@martynamolak offered to review the package. Thank you!

martynamolak commented:
@Thanks @Tlkhi for submitting this!
Here is my comments:

janno file:
1) Relation_To column should probably comprise Poseidon_ID of the related individuals, not the Alternative_IDs. I realise this is a bit tricky because the relation will (should) be the same regardless of the instance of a given individual's reanalysis. @nevrome , should we enforce this? (e.g. "I23568.AG" has Relation_To: "I23655" rather than to "I23655.AG")
2) Please change "ENA: PRJEB81467" to "PRJEB81467" in the Genetic_Source_Accession_IDs field
3) The Poseidon_ID and Group_Name have the ".SG", ".AG" and ".TW" suffixes that are, as far as I could find, not present in the original paper. I of course see the value of the suffixes but also Poseidon's policy is to keep the labels concordant with the original paper. That's something that should probably be discussed throughout the Poseidon dev team, @nevrome, @TCLamnidis, @stschiff; Anyway, I think at current stage it's probably better to leave the suffixes in; ie.: as currently is labelled in the package.
4) Y haplogroups are provided in the hierarchichal system (ISOGG) rather than the terminal-SNP-based system (Yfull) as required/encouraged by Poseidon. The original paper provides both so they should be easy enough to replace.
5) some individuals are merges of several libraries; some of these libraries differ in strandedness and damage removal procedure (e.g. I5124 is 4 libraries: ds.half,ds.half,ss.USER,ss.USER in original paper while in the package it's coded solely as: ds, half) - please verify all the samples and correct where necessary
6) It would be nice to also include contamination estimates in the janno file. The original supplementary table provides HapConX and ANGSD estimates. Unfortunately they would have to be transposed from ranges to the mean+stderr format for Poseidon. Also damage rate column could be added.

All the other files in the package seem ok as far as I could see.

Tlkhi commented:
> @thanks @Tlkhi for submitting this! Here is my comments:
> 
> janno file:
> 
> 1. Relation_To column should probably comprise Poseidon_ID of the related individuals, not the Alternative_IDs. I realise this is a bit tricky because the relation will (should) be the same regardless of the instance of a given individual's reanalysis. @nevrome , should we enforce this? (e.g. "I23568.AG" has Relation_To: "I23655" rather than to "I23655.AG")
> 2. Please change "ENA: PRJEB81467" to "PRJEB81467" in the Genetic_Source_Accession_IDs field
> 3. The Poseidon_ID and Group_Name have the ".SG", ".AG" and ".TW" suffixes that are, as far as I could find, not present in the original paper. I of course see the value of the suffixes but also Poseidon's policy is to keep the labels concordant with the original paper. That's something that should probably be discussed throughout the Poseidon dev team, @nevrome, @TCLamnidis, @stschiff; Anyway, I think at current stage it's probably better to leave the suffixes in; ie.: as currently is labelled in the package.
> 4. Y haplogroups are provided in the hierarchichal system (ISOGG) rather than the terminal-SNP-based system (Yfull) as required/encouraged by Poseidon. The original paper provides both so they should be easy enough to replace.
> 5. some individuals are merges of several libraries; some of these libraries differ in strandedness and damage removal procedure (e.g. I5124 is 4 libraries: ds.half,ds.half,ss.USER,ss.USER in original paper while in the package it's coded solely as: ds, half) - please verify all the samples and correct where necessary
> 6. It would be nice to also include contamination estimates in the janno file. The original supplementary table provides HapConX and ANGSD estimates. Unfortunately they would have to be transposed from ranges to the mean+stderr format for Poseidon. Also damage rate column could be added.
> 
> All the other files in the package seem ok as far as I could see.

Thank you for your comments!

1. I’m waiting for @nevrome for now - it might look a bit complicated since there is no Individual_ID column yet


2. Fixed


3. I think we should keep the suffixes


4. Replaced ISOGG format with SNP-terminal based format - fixed


5. I will mark them as mixed and note the UDG types (e.g., half; USER) in the Note column


6. Added the Damage column, adding a Contamination column seems a bit challenging

stschiff commented:
For what it's worth, I am OK with the suffixes... I think we have no official policy on this, and since the original authors haven't submitted the package, but @Tlkhi has, they get to decide. I believe since it's a Boston Paper, AADR-like suffixes make sense.

Stephan

Tlkhi commented:
fixed the issues
